.evg.yml

```diff
@@ -150,8 +149,17 @@ tasks:
             export PATH=$HOME/bin:$PATH
             export AWS_ACCESS_KEY_ID=${sdks_aws_key}
             export AWS_SECRET_ACCESS_KEY=${sdks_aws_secret}
-            aws s3 cp stitch-js-sdk/docs s3://stitch-sdks/js/docs/${revision} --recursive --acl public-read
-            aws s3 cp stitch-js-sdk/docs s3://stitch-sdks/js/docs/${branch_name} --recursive --acl public-read
+            
+            if [ "${is_patch|}" = "true" ]; then
```

Comment:
i think it could occasionally be useful when changing + reviewing docs, e.g. https://jira.mongodb.org/browse/STITCH-469. can remove though